installer/roles/local_docker/tasks/compose.yml

```diff
@@ -36,6 +36,12 @@
   register: awx_secret_key
 
 - block:
+    - name: Remove AWX containers before migrating postgres so that the old postgres container does not get used
```

Comment:
Removing the old postgres container here prevents docker-compose from using it, as it satisfies the `depends_on` for the task container.  The result is that postgres:12 is used every time, regardless of what other postgres containers are available.